flow.py

```diff
@@ -0,0 +1,122 @@
+class Flow(object):
+    """ A flow class that represents active connections between
+    hosts and routers.
+
+    Args:
```

Comment:
No need for args for constructor in this case. Just put these in the attributes since these are all attributes.